README.md

```diff
@@ -40,12 +40,12 @@ cat input.css | postcss [OPTIONS] > output.css
 |Name|Type|Default|Description|
 |:---|:--:|:-----:|:----------|
 |`-d, --dir`|`{String}`|`undefined`|Output Directory|
+|`--base`|`{String}`|`undefined`|Use together with `--dir` for keeping directory structure.|
```

Comment:
> I didn't add a short hand because it's only usable together with -d, does that make sense?

¯\_(ツ)_/¯ It's preference, doesn't hurt to have both imho